requirements/features-executable.yaml

```diff
@@ -412,10 +412,14 @@ requirements:
       - executable
       - feature
       - linux
+    links:
+      - name: Understanding Real-time programming
+        url: https://docs.ros.org/en/iron/Tutorials/Demos/Real-Time-Programming.html#plotting-results
     checks:
       - name: Check `rttest_plot`
         try:
-          - stdin: ros2 run rttest rttest_plot
+          - stdin: pendulum_demo -f pendulum_demo_results
```

Comment:
So, I'm a bit conflicted here.  On the one hand, this command is absolutely correct for Iron.  On the other hand, for Rolling we ended up changing this (via https://github.com/ros2/demos/pull/624) to `ros2 run pendulum_control pendulum_demo -f pendulum_demo_results`.  So if we end up copying this for Jazzy (which we almost certainly will), this will no longer be correct.

@Yadunund is going to do some work here to split these configurations out of this repository.  I would suggest we hold off on this one until we do that, at which point we can add in a Jazzy set of configuration.  And then we can merge this one into the Iron configuration, and the updated one into the Jazzy configuration.

Reply:
Ok sounds good!